3.analysis/scripts/nbconverted/2.gene_set_enrichment_analysis.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 # coding: utf-8
 
-# In[1]:
+# In[33]:
```

Comment:
Please make sure your notebooks are run sequentially. Otherwise, there is no way to confirm data/results fidelity across the notebook.

Reply:
I will review the full PR after you address this item! :)